python/graphstorm/gsf.py

```diff
@@ -476,28 +476,34 @@ def create_builtin_node_decoder(g, decoder_input_dim, config, train_task):
     dropout = config.dropout if train_task else 0
     if config.task_type == BUILTIN_TASK_NODE_CLASSIFICATION:
         if not isinstance(config.num_classes, dict):
-            decoder = EntityClassifier(decoder_input_dim,
-                                       config.num_classes,
-                                       config.multilabel,
-                                       dropout=dropout,
-                                       norm=config.decoder_norm,
-                                       use_bias=config.decoder_bias)
+
+            decoder_output_dim = config.num_classes
             if config.class_loss_func == BUILTIN_CLASS_LOSS_CROSS_ENTROPY:
                 loss_func = ClassifyLossFunc(config.multilabel,
                                              config.multilabel_weights,
                                              config.imbalance_class_weights)
             elif config.class_loss_func == BUILTIN_CLASS_LOSS_FOCAL:
-                assert config.num_classes == 1, \
+                assert config.num_classes in [1, 2], \
```

Comment:
Add a comment here:
`For backward compatibility, we allow the num_classes to be 1. We suggest users to set it to 2. `